seatunnel-connectors-v2/connector-file/connector-file-sftp/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/sftp/source/SftpFileSource.java

```diff
@@ -72,7 +74,7 @@ public void prepare(Config pluginConfig) throws PrepareFailException {
         if (fileFormat == FileFormat.ORC || fileFormat == FileFormat.PARQUET) {
             throw new FileConnectorException(
                     CommonErrorCode.ILLEGAL_ARGUMENT,
-                    "Sftp file source connector only support read [text, csv, json] files");
+                    "Sftp file source connector does not support read [orc, parquet] files");
```

Comment:
Why change this message?

Reply:
I thought it was more in line with the intent of the code cause I added the `xml` file type, so I changed here, should I change back? Just like `Sftp file source connector only support read [text, csv, json, (xml)] files);`

Reply:
It was unnecessary change, I thought it's better to change back. And if you want to keep it in line, why not use a static constant like 'static List<String> SUPPORT_TYPE= Lists.asList("text", "csv", "json", "xml")'?

Reply:
Okay, I'll change it back